run-health-checks.sh

```diff
@@ -77,7 +77,7 @@ if [ -z "$CONF_FILE" ]; then
     echo "No custom conf file specified, detecting VM SKU..."
     SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text" | sed 's/Standard_//')
```

Comment:
Should we go ahead and do the lower case before sed? Just in case something goes wrong with the metadata command at some point?

Reply:
That is what the PR for rescale is. I can implement the change. I would need to ask rescale to close their PR.

Reply:
I will address on next PR or rescale PR